embedded-graphics/src/primitives/circle/styled_iterator.rs

```diff
@@ -0,0 +1,154 @@
+use crate::{
+    drawable::Pixel,
+    pixelcolor::PixelColor,
+    primitives::circle::{diameter_to_threshold, distance_iterator::DistanceIterator, Circle},
+    style::{PrimitiveStyle, Styled},
+};
+
+/// Pixel iterator for each pixel in the circle border
```

Comment:
> All Styled... iterators need a new comment.

Not entirely sure what you mean by this. Do they need tweaking or expanding or something else?

> We don't use "border" elsewhere IIRC and it doesn't describe the purpose very well IMO.

True, I'll fix this.

Reply:
> Not entirely sure what you mean by this. Do they need tweaking or expanding or something else?

The main change was described in the following sentence, but IMO they could also be expanded a bit. IMO the comment should describe what a `StyledIterator` is and how it can be created.